internal/operands/vm-console-proxy/resources.go

```diff
@@ -0,0 +1,226 @@
+package vm_console_proxy
```

Comment:
The resources for `vm-console-proxy` are loaded from a yaml file, so they should not be hardcoded here.
I think you can remove this file.

Reply:
Done, thanks!